src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -123,13 +123,18 @@ export default defineComponent({
       type: String,
       default: null
     },
+    currentPlaybackRate: {
+      type: Number,
+      default: () => 1.0
+    },
   },
   emits: [
     'error',
     'loaded',
     'ended',
     'timeupdate',
-    'toggle-theatre-mode'
+    'toggle-theatre-mode',
+    'current-playback-rate-update'
```

Comment:
Slightly shorter event name with the same meaning.
```suggestion
    'playback-rate-updated'
```